index.js

```diff
@@ -12,21 +12,20 @@ module.exports.announceList = [
   [ 'wss://tracker.fastcast.nz' ]
 ]
 
-var bencode = require('bencode')
-var BlockStream = require('block-stream2')
-var calcPieceLength = require('piece-length')
-var corePath = require('path')
-var extend = require('xtend')
-var FileReadStream = require('filestream/read')
-var flatten = require('flatten')
-var fs = require('fs')
-var isFile = require('is-file')
-var junk = require('junk')
-var MultiStream = require('multistream')
-var once = require('once')
-var parallel = require('run-parallel')
-var sha1 = require('simple-sha1')
-var stream = require('readable-stream')
+const bencode = require('bencode')
```

Comment:
Could you please make this the first line and put the exports at the end?

Reply:
bara bing bara boom